lib/vparse.c

```diff
@@ -1042,10 +1045,55 @@ EXPORTED void vparse_delete_entries(struct vparse_card *card, const char *group,
             entry->next = NULL; /* so free doesn't walk the chain */
             _free_entry(entry);
         }
+        else {
+            if (entry->group && props_using_label_counts) {
+                /* Count properties associated with an Apple label in this group */
+                uintptr_t count = (uintptr_t) hash_lookup(entry->group,
+                                                          props_using_label_counts);
+
+                if (strcasecmpsafe(entry->name, APPLE_LABEL_PROPERTY)) {
+                    /* NOT a label property, count it */
+                    count++;
+                }
+                hash_insert(entry->group, (void *) count, props_using_label_counts);
+            }
+
+            entryp = &((*entryp)->next);
+        }
+    }
+}
+
+EXPORTED void vparse_delete_entries(struct vparse_card *card,
```

Comment:
Would there ever be a reason to only call `vparse_delete_entries` instead of deleting entries and labels?

Reply:
Probably not.  I will fold this into one function